css/gallery.css

```diff
@@ -1,107 +1,130 @@
 /*class for the section of gallery*/
+.gallery-item::before {
+  content: " ";
+  display: block;
+  padding-top: 100%;
+}
 .gallery {
-  width: 635px;
-  height: 88px;
   font-family: Roboto;
   font-size: 16px;
   color: black;
-  padding: 60px 0;
+  padding-right: 20px;
 }
 .gallery__info {
-  top: 0;
-  left: 0;
-  margin-right: 150px;
-  margin-left: 150px;
-  margin-bottom: 17px;
-}
-.gallery__text {
-  padding-bottom: 17px;
-  margin-block-start: 0px;
-  margin-block-end: 0px;
+  display: flex;
+  flex-flow: row wrap;
+  margin-left: 20px;
+  margin-right: 20px;
+  margin-bottom: 30px;
 }
 .gallery__items {
   display: flex;
   flex-flow: row wrap;
   justify-content: center;
-}
-.gallery__items :nth-child(4n) {
-  margin-right: 0px;
+  min-width: 320px;
+  margin-left: 20px;
 }
 .gallery-item {
   background-position: center;
-  background-size: cover;
   position: relative;
-  width: 227px;
-  height: 227px;
-  margin-right: 17px;
-  margin-bottom: 17px;
   box-shadow: 0px 0px 2px 0px rgba(0, 0, 0, 0.75);
   overflow: hidden;
+  flex-basis: 41%;
+  flex-grow: 1;
+  margin-right: 20px;
+  margin-bottom: 20px;
 }
 .gallery-item__images {
   width: 100%;
   height: 100%;
   background-size: cover;
-  transition: 0.1s;
-  -webkit-transition: all 0.1s ease;
-  -moz-transition: all 0.9s ease; /* Firefox */
-  -o-transition: all 0.9s ease; /* IE 9 */
-  -ms-transition: all 0.9s ease; /* Opera */
-}
-.gallery-item:hover .gallery-item__images {
-  -webkit-transform: scale(1.25);
-  -moz-transform: scale(1.25);
-  -ms-transform: scale(1.25);
-  -o-transform: scale(1.25);
-  transform: scale(1.25);
-}
-/* overlay gray for the images */
-.gallery-item__overlay {
   position: absolute;
-  bottom: 0%;
-  left: 0%;
-  width: 0%;
-  height: 0%;
-  background-color: rgba(0, 0, 0, 0.541);
-  overflow: hidden;
-  vertical-align: middle;
+  top: 0;
+  left: 0;
 }
-.overlay__image {
-  display: block;
-  width: 50px;
-  height: 50px;
-  margin: auto;
-  margin-top: 89px;
+.gallery-item__overlay {
+  display: none;
 }
-.gallery-item:hover .gallery-item__overlay {
-  width: 227px;
-  height: 227px;
-  cursor: pointer;
+.gallery__items :nth-child(2n) {
+  margin-right: 0;
 }
 /* start media queries */
-@media screen and (max-width: 960px) {
+@media screen and (min-width: 960px) {
+  .gallery {
+    width: 635px;
+    height: 88px;
```

Comment:
this height rule, does it serve some purpose, it seems it would cause problems

Reply:
reviewing it does nothing